amico/models.py

```diff
@@ -1,18 +1,13 @@
 from __future__ import absolute_import, division, print_function
 
 import numpy as np
-import numpy.matlib as matlib
 import scipy
-from os.path import exists, join as pjoin
-from os import remove
-import subprocess
-import tempfile
+from os.path import join as pjoin
 import amico.lut
 from tqdm import tqdm
-from dipy.core.gradients import gradient_table
-from dipy.sims.voxel import single_tensor
 import abc
-from amico.util import PRINT, LOG, NOTE, WARNING, ERROR, get_verbose
+from amico.util import PRINT, ERROR, get_verbose
+from amico.synthesis import Stick, Zeppelin, Ball, CylinderGPD, SphereGPD, Astrosticks, NODDISignal
```

Comment:
Can we find a better alternative for the `NODDIsignal` name?

Reply:
What about three separate classes for each NODDI compartment?
e.g. `NODDIIntraCellular()`, `NODDIExtraCellular()` and `NODDIIsotropic`

Reply:
This can be a suitable option!

Reply:
commit 733ab16ad6e050ca44ba56e8cd154c82c45de922